apps/src/gamelab/AnimationTab/ListItemButtons.jsx

```diff
@@ -77,6 +92,12 @@ ListItemButtons.propTypes = {
   onLoopingChanged: React.PropTypes.func.isRequired,
   looping: React.PropTypes.bool.isRequired,
   onFrameDelayChanged: React.PropTypes.func.isRequired,
-  frameDelay: React.PropTypes.number.isRequired
+  frameDelay: React.PropTypes.number.isRequired,
+  allAnimationsSingleFrame: React.PropTypes.bool.isRequired
 };
-module.exports = Radium(ListItemButtons);
+
+module.exports = connect(function propsFromStore(state) {
+  return {
+    allAnimationsSingleFrame: state.pageConstants.allAnimationsSingleFrame
```

Comment:
Likewise here - maybe we just pass the frame count down into this component?  I wonder if we should hide the speed slider and looping toggle for single-frame animations, even when animations are enabled?